<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Notepad</class>
 <widget class="QMainWindow" name="Notepad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>21</y>
      <width>681</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionsave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionFont"/>
    <addaction name="actionItalic"/>
    <addaction name="actionBold"/>
    <addaction name="actionUnder_line"/>
    <addaction name="separator"/>
    <addaction name="actionColor"/>
    <addaction name="actionBackground_Color"/>
    <addaction name="actionBackground_Color_Editor"/>
   </widget>
   <widget class="QMenu" name="menuabout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_Notepad"/>
   </widget>
   <widget class="QMenu" name="menuSearch">
    <property name="title">
     <string>Zoom</string>
    </property>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
   </widget>
   <widget class="QMenu" name="menuAlignment">
    <property name="title">
     <string>Alignment</string>
    </property>
    <addaction name="actionCenter"/>
    <addaction name="actionRight"/>
    <addaction name="actionLeft"/>
    <addaction name="actionJustify"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSearch"/>
   <addaction name="menuAlignment"/>
   <addaction name="menuabout"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionsave"/>
   <addaction name="actionSave_As"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionCenter"/>
   <addaction name="actionJustify"/>
   <addaction name="actionRight"/>
   <addaction name="actionItalic"/>
   <addaction name="actionBold"/>
   <addaction name="actionUnder_line"/>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_out"/>
   <addaction name="actionColor"/>
   <addaction name="actionFont"/>
   <addaction name="actionPrint"/>
   <addaction name="actionAbout_Notepad"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="texteditorimages.qrc">
     <normaloff>:/new/TextEditor/Images/add.png</normaloff>
     <normalon>Images/add.png</normalon>:/new/TextEditor/Images/add.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="texteditorimages.qrc">
     <normaloff>:/new/TextEditor/Images/door_open.png</normaloff>:/new/TextEditor/Images/door_open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="icon">
    <iconset>
     <normalon>:/new/TextEditor/Images/page_save.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset>
     <normalon>:/new/TextEditor/Images/script_save.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normalon>:/new/TextEditor/Images/arrow_undo.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="texteditorimages.qrc">
     <normaloff>:/new/TextEditor/Images/arrow_redo.png</normaloff>:/new/TextEditor/Images/arrow_redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="texteditorimages.qrc">
     <normaloff>:/new/TextEditor/Images/cut.png</normaloff>:/new/TextEditor/Images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <normalon>:/new/TextEditor/Images/page_copy.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset>
     <normalon>:/new/TextEditor/Images/page_white_paste.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset>
     <normalon>:/new/TextEditor/Images/printer.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="icon">
    <iconset>
     <normalon>:/new/TextEditor/Images/color_swatch.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionBackground_Color">
   <property name="icon">
    <iconset resource="texteditorimages.qrc">
     <normaloff>:/new/TextEditor/Images/backclr.jpg</normaloff>:/new/TextEditor/Images/backclr.jpg</iconset>
   </property>
   <property name="text">
    <string>Highlight</string>
   </property>
  </action>
  <action name="actionBackground_Color_Editor">
   <property name="icon">
    <iconset>
     <normalon>:/new/TextEditor/Images/full.jpg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Background Color Editor</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset>
     <normalon>:/new/TextEditor/Images/font.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionAbout_Notepad">
   <property name="icon">
    <iconset>
     <normalon>:/new/TextEditor/Images/About_Icon_1.jpg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>About Notepad</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="icon">
    <iconset>
     <normalon>:/new/TextEditor/Images/zoom_in.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="icon">
    <iconset>
     <normalon>:/new/TextEditor/Images/zoom_out.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="icon">
    <iconset>
     <normalon>:/new/TextEditor/Images/text_italic.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset>
     <normalon>:/new/TextEditor/Images/text_bold.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionUnder_line">
   <property name="icon">
    <iconset>
     <normalon>:/new/TextEditor/Images/text_underline.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Under line</string>
   </property>
  </action>
  <action name="actionCenter">
   <property name="icon">
    <iconset>
     <normalon>:/new/TextEditor/Images/text_align_center.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Center</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="icon">
    <iconset>
     <normalon>:/new/TextEditor/Images/text_align_right.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="icon">
    <iconset>
     <normalon>:/new/TextEditor/Images/text_align_left.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionJustify">
   <property name="icon">
    <iconset>
     <normalon>:/new/TextEditor/Images/text_align_justify.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Justify</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normalon>:/new/TextEditor/Images/Exit.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="texteditorimages.qrc"/>
 </resources>
 <connections/>
</ui>
